features/networking/operator.feature

```diff
@@ -168,16 +147,8 @@ Feature: Operator related networking scenarios
     Given the status of condition "Progressing" for network operator is :True
     """
 
-    And I wait up to 60 seconds for the steps to pass:
-    """
-    When I run the :get admin command with:
-      | resource      | machineset                |
-      | resource_name | <%= cb.machineset %>      |
-      | n             | openshift-machine-api     |
-      | template      | {{.status.readyReplicas}} |
-    Then the expression should be true> @result[:response].to_i == cb.desired_replicas
-    """
-
+    Then the step should succeed
```

Comment:
What's this line for ?

Reply:
Delete this step, it's not useful. thanks